app.js

```diff
@@ -40,58 +62,78 @@ storeSalesDataEl.appendChild(timeRowEl);
-    var cookieCountEl = document.createElement('td');
-    cookieCountEl.textContent = stores[storeIndex].hourlyCookieCount[cookieData];
-    tableRowEl.appendChild(cookieCountEl);
+function creatingHeaderRow () {
+  for (var cell = 0; cell < firstAndPike.businessHours.length; cell++) { // Needed time from first store to display hours.
+    var timeHour = document.createElement('td');
+    timeHour.textContent = firstAndPike.businessHours[cell];
+    timeRowEl.appendChild(timeHour);
   }
+}
+creatingHeaderRow();
 
-  var dailyLocationTotalEl = document.createElement('td');
-  dailyLocationTotalEl.textContent = firstAndPike.totalCountForDay;
-  tableRowEl.appendChild(dailyLocationTotalEl);
+for (var storeIndex = 0; storeIndex < stores.length; storeIndex++) {
+  stores[storeIndex].renderRow();
 }
 
-var sectionEl = document.getElementById('cookieTableSection');
+var sectionEl = document.getElementById('cookie_table_section');
 sectionEl.appendChild(storeSalesDataEl);
-
-// var tableEl = document.createElement('table');  //Step 1 - Create Element
+// console.log('----------------EVENTS LISTENERS------------------');
 //
-// for (var i = 0; i < stores.length; i++){
-//   var currentStore = store[i];
+// var storeFormEl = document.getElementById('new-store-form');
 //
-//   var rowEl = document.createElement('tr'); //Step 3 - Create elements to loop through.
-//   tableEl.appendChild(nameEl);
+// storeFormEl.addEventListener('submit', handleSubmit);
 //
-//   var nameEl = document.createElement('th'); // 'th' tag to create a heading
-//   nameEl.textContent = currentStore.name;
-//   rowEl.appendChild(nameEl);
+// function handleSubmit(event){
+//   event.preventDefault();
+//   event.stopPropagation();
+//   var name = event.target.cookieStoreName.value;
+//   var minCustomers = parseInt(event.target.minCust.value);
+//   var masCustomers = parseInt(event.target.maxCust.value);
+//   var avgCookies = parseInt(event.target.avgCookies.value);
 //
-//   var minCustEl = document.createElement('td');
-//   minCustEl.textContent = currentStore.minCustomers;
-//   rowEl.appendChild(minCustEl);
+//   // console.log(name);
+//   // console.log(minCustomers);
+//   // console.log(maxCustomers);
 //
-//   var maxCustEl = document.createElement('td');
-//   maxCustEl.textContent = currentStore.maxCustomers;
-//   rowEl.appendChild(maxCustEl);
+//   var store = new CookieStore(name,minCustomers, maxCustomers, avgCookies);
 //
-//   var avgCookiesEl = document.createElement('td');
-//   avgCookiesEl.textContent = currentStore.avgCookies;
-//   rowEl.appendChild(avgCookiesEl);
+//   stores.push(store);
 //
+//   console.log(store);
 // }
-
-// document.body.appendChild(tableEl); // Step 2 - don't forgot to add element to the document
+//
+// // var tableEl = document.createElement('table');  //Step 1 - Create Element
+// //
+// // for (var i = 0; i < stores.length; i++){
```

Comment:
If you're done with all this code and aren't actively using it for reference anymore, feel free to delete it. You can always go back and look at it in your git history.